Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -1604,6 +1741,29 @@ def convert_start_fetch_to_milliseconds(fetch_start_time: str):
         raise ValueError(f'Invalid first_fetch format: {fetch_start_time}')
     return int(date.timestamp() * 1000)
 
+def convert_dict_values_string_to_number(obj: dict) -> dict:
+    clone = { **obj }
+    for key, value in obj.items():
+        if isinstance(value, str) and value.isnumeric():
+            clone[key] = int(value)
+    return clone
+
+def convert_to_actual_values_recursive(input_dict: dict) -> dict[str, Any]:
+    output_dict = {}
+    for key, value in input_dict.items():
+        if isinstance(value, dict):
```

Comment:
what if there is a key that is an array? we will miss the recursion there

Reply:
done